### DISCUSSION TITLE
Latest version uses async scripts instead of defer with app router

### DISCUSSION THREAD

**Discussion by AdarshKonchady** [vercel/next.js#58982] [App Router]
I was doing some performance analysis on my website and found my LCP element was blocked by JS execution. Digging more, noticed with latest version of NextJS, scripts (application server scripts generated by NextJS) are loaded async instead of defer. Is there any way to change this behavior? 

Found this old issue where the default was changed from async to defer but looks like the default was changed back again ? 
https://github.com/vercel/next.js/discussions/11120

Tested with NextJS 14.0.3.

**Comment by MISRV001** [comment]
@ijjk  / @timneutkens   /  NextJs Community -  Is this expected to fix soon as impacting application performances.

**Comment by markoleavy** [comment]
Same problem here. 
Isn't that the opposite of the expected behaviour on SSG pages, where static html should be rendered before loading, parsing and executing js?

**Comment by markoleavy** [comment]
@AdarshKonchady it looks like version 13.5.4 has better performance. Still using async in head scripts, but switching back from 14.1.0 my doc head passed from having more than 30 script tags to 3, and my LCP from 5-6 sec to 2.7s.  More details on [57989](https://github.com/vercel/next.js/discussions/57989).
